backend/src/Configuration.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+
+class Configuration:
+  DEFAULT_VALUES : dict = {
+    "SMTP_SERVER" : "smtp.gmail.com",
+    "SMTP_PORT"   : 465
+  }
+
+  def __init__(self):
+    self.CONF_FILE_LOCATION = './conf.json' if 'SYNCER_CONF' not in os.environ.keys() else os.environ['SYNCER_CONF']
```

Comment:
set self.CONF_FILE_LOCATION: str 
```suggestion
    self.CONF_FILE_LOCATION: str = './conf.json' if 'SYNCER_CONF' not in os.environ.keys() else os.environ['SYNCER_CONF']
```